extensions/amp-ad/0.1/amp-ad-api-handler.js

```diff
@@ -122,20 +135,31 @@ export class AmpAdApiHandler {
       this.iframe_.style.visibility = 'hidden';
     }
     listenForOnce(this.iframe_, 'render-start', () => {
+      perf.tick('e_ar');
```

Comment:
yes, I described it incorrectly to @zhouyx. I'll get with her this morning to fix